app/views/sidebar/_related.html.erb

```diff
@@ -77,7 +77,7 @@
   <% if @maps && @maps.length > 0 && params[:controller] != "search" %>
 
     <br />
-    <div style="width:100%;margin-left:0;height:300px;" id="map"></div>
+    <div style="width:100%;margin-left:0;height:300px;"></div>
```

Comment:
This does move the map div to the partial, but it leaves the styled div here, unused. Can you remove this line entirely, and move the style attributes to the partial `_leaflet.html.erb`? Thanks!